datasets/gov/palangos_gintaras/pacientai.csv

```diff
@@ -0,0 +1,54 @@
+id,dataset,resource,base,model,property,type,ref,source,prepare,level,access,uri,title,description
+,datasets/gov/palangos_gintaras/pacientai,,,,,,,,,,,,,
+,,,,,,prefix,spinta,,,,,https://github.com/atviriduomenys/spinta/issues/,,
+,,,,,,,manifest,,,,,https://github.com/atviriduomenys/manifest/issues/,,
+,,,,,,,vadovas,,,,,https://atviriduomenys.readthedocs.io/,,
+,,,,,,,,,,,,,,
+,,,,Ligonis,,,vda_id,,,4,,,Vaikų reabilitacijos sanatorijos „Palngangos gintaras“ pacientų sąrašas,Vaikų reabilitacijos sanatorijos „Palngangos gintaras“ pacientų sąrašas. Duomenys pateikiami už 2016-2023 metus
+,,,,,vda_id,string,,,,4,open,,Identifikacinis numeris,
+,,,,,paciento_id,string,,,,,open,,Paciento identifikacinis numeris,
+,,,,,gimimo_metai,date,D,,,,open,,Paciento gimimo metai,
+,,,,,lytis,string,,,,,open,,Paciento lytis,
+,,,,,neigalumo_tipas,string,,,,,open,,Paciento neigalumno tipas,
+,,,,,,,,,,,,,,
+,,,,PacientoRegistracija,,,vda_id,,,4,,,Vaikų reabilitacijos sanatorijos „Palngangos gintaras“ pacientų registracijų sąrašas,Vaikų reabilitacijos sanatorijos „Palngangos gintaras“ pacientų registracijų sąrašas. Duomenys pateikiami nuo 2010 metų ir iki dabar.
+,,,,,vda_id,string,,,,4,open,,Identifikacinis numeris,
+,,,,,registracijos_nr,string,,,,4,open,,Paciento registracijos identifikacinis numeris,
+,,,,,paciento_id,ref,/datasets/gov/palangos_gintaras/pacientai/Ligonis[paciento_id],,,4,open,,Paciento identifikacinis numeris,
+,,,,,atvykimo_data,date,D,,,4,open,,Paciento atvykimo data,
+,,,,,isvykimo_data,date,D,,,4,open,,Paciento išvykimo data,
+,,,,,atvykimo_registracijos_data,date,S,,,4,open,,Paciento atvykimo registracijos data,
+,,,,,tlk_kodas,string,,,,4,open,,Teritorinės ligonių kasos kodas,
+,,,,,tlk_pav,string,,,,4,open,,Teritorinės ligonių kasos pavadinimas,
+,,,,,lovadieniu_skaicius,integer,,,,4,open,,Lovadienių skaičius,
```

Comment:
Gali nurodyti, tikslumą - lovadienių skaičius turbūt dienomis, tada "d" siūlyčiau naudoti

Reply:
Laiko tikslumas veikai tik su date arba datetime duomenu tipais. Su integer taip daryti negalima

Reply:
Taip, Tu teisingai sakai. Tik aš turiu omenyje laiko vienetą, kai nurodoma trukmė, dėl to ir d mažoji raidė (https://atviriduomenys.readthedocs.io/dsa/vienetai.html aprašyta skyrelyje "Laiko vienetai").